dev/releases/release_notes.py

```diff
@@ -165,16 +167,33 @@ def get_pr_list(date: str, extra: str) -> List[Dict[str, Any]]:
 def pr_to_md(pr: Dict[str, Any]) -> str:
     """Returns markdown string for the PR entry"""
     k = pr["number"]
-    title = pr["title"]
-    return f"- [#{k}](https://github.com/oscar-system/Oscar.jl/pull/{k}) {title}\n"
+    if has_label(pr, 'release notes: use body'):
+        mdstring = pr['body'].replace("- ", f"- [#{k}](https://github.com/oscar-system/Oscar.jl/pull/{k}) ")
```

Comment:
Won't this also replace any `- ` that occurs in the middle of a description? Might be better to use a regex anchor, i.e., something like
```suggestion
        mdstring = pr['body'].replace(r"^- ", f"- [#{k}](https://github.com/oscar-system/Oscar.jl/pull/{k}) ")
```

Reply:
I don't know why, but as of this moment, I am unwilling to use regx. I have tried to make an alternate solution: replace only the first occurrence in string.

I think regx (dash only at first position) might be a stronger solution, but my most recent changes might be enough in this case.

I will change it to regx if you think its better in any case.

Reply:
Well, there is a concrete bug that can relatively easily occur with your current code, so yeah, I really think you should either use the regex here, or use some other approach that ensures the dash is at the start of a line (e.g. you could also match for `"\n- "` and replace by `"\n- ..."`)

Reply:
Resolved with regx